src/components/timeout-dialog/timeout-dialog.js

```diff
@@ -212,17 +211,17 @@ function TimeoutDialog($module, $sessionActivityService) {
       let minutes; let
         visibleMessage;
       if (counter < 60) {
-        visibleMessage = `${counter} ${settings.properties[counter !== 1 ? 'seconds' : 'second']}`;
+        visibleMessage = `${counter} ${settings.properties[counter !== 1 ? 'seconds' : 'second']}.`;
       } else {
         minutes = Math.ceil(counter / 60);
-        visibleMessage = `${minutes} ${settings.properties[minutes === 1 ? 'minute' : 'minutes']}`;
+        visibleMessage = `${minutes} ${settings.properties[minutes === 1 ? 'minute' : 'minutes']}.`;
       }
       return visibleMessage;
     };
 
     const getAudibleHumanText = (counter) => {
       const humanText = getHumanText(roundSecondsUp(counter));
-      const messageParts = [settings.message, ' ', humanText, '.'];
+      const messageParts = [settings.message, ' ', humanText];
```

Comment:
@TimothyFothergill what was the reasoning for removing it from the audible? without a period sometimes screen readers won't pause in between bits of text I think

Reply:
(not sure if it applies here at all, just double checking reason for changes that I wasn't sure of source of)

Reply:
Ah I remember now:
humanText now contains the . - When it does getHumanText, it includes the .

Reply:
https://github.com/hmrc/hmrc-frontend/blob/9bacc29f84412326fc237c4cf9a9682f3c397f26/src/components/timeout-dialog/timeout-dialog.js#L211-L219

Reply:
aha! so it's in both still but added in a different way :legend: thank you Tim!